x-pack/plugins/security_solution/server/lib/telemetry/sender.ts

```diff
@@ -57,6 +59,7 @@ export class TelemetryEventsSender {
   private intervalId?: NodeJS.Timeout;
   private isSending = false;
   private queue: TelemetryEvent[] = [];
+  private usageLabelPrefix: string[] = ['security_telemetry', 'sender'];
```

Comment:
It might be more idiomatic TS to make this a constant.

Reply:
Ok